### DISCUSSION TITLE
Crash reports without any information

### DISCUSSION THREAD

**Discussion by frankystone** [widelands/widelands#6249] [General]
Are [such](https://github.com/widelands/widelands/issues/6248) reports helpful for you developers? There were more in the past.

From my point of view they are useless, because some information is missing and should immediately be closed, probably with a note that the bug report template is there on purpose. I know this is somewhat impolite, but throwing out such bugreports is also impolite, imho.

I also want to suggest to add a question about the used add-ons to the bug report template.

**Comment by matthiakl** [comment]
> Are such reports helpful for you developers?

Well, without context most often not because there is no hint what caused the crash.

> From my point of view they are useless, because some information is missing and should immediately be closed, probably with a note that the bug report template is there on purpose. 

We can define a standard response for such cases, something like:

@\<user> you ignored our issue template and important information are missing. 
If you fill the template you may reopen the issue.

> I also want to suggest to add a question about the used add-ons to the bug report template.

+1
This could be a bullet in the "Version" list. And maybe also a distinct section for logs or crash reports as hint, that the report on its own is not sufficient.

**Comment by tothxa** [comment]
I think we should expand the instruction in the crash report window to ask users to use the bug template properly to also describe what was going on in the game and what they did just before the crash, and if possible, attach the replay too. Also make it clear that the crash log does not do as a bug report in itself, it belongs to the additional context section instead.

As for expanding the template, indeed, addons are an important info. We could also add headings for replay, last save and crash log in addition to addons.

I think I'll create a PR with these. :)
